docs/developer/images.md

```diff
@@ -102,27 +151,17 @@ file to the project. Drag your palette file ( `.gpl`, `.txt`, or `.hex` format)
 onto the page to have it appear in the palette dropdown beneath each image entry. If your
 sprites come out looking weird, you should double check the palette you're using.
 
-https://lospec.com/palette-list is an amazing resource for finding color palettes.
+As an example, let's say you uploaded an image file `mosaic.png` with a palette called `cool-colors.txt`. In the asset tool you select the `Cool Colors` palette to use with the image. The exported ``assets.ts`` file contains a custom image that uses the `Cool_Colors` palette.
 
-If you already have a palette that isn't in one of the above formats, you can easily create a
-`.hex` file by hand. Simply put each color's hex string on a separate line and save a file
-with the `.hex` extension. For example:
+```typescript
+// assets.ts
 
+namespace projectImages {
+    //% fixedInstance
+    export const mosaic = image.ofBuffer(hex`e4101000dddddddddddddddd5d55454444eeeede5d55646666e4eede5d4566666646eede5d6466666666e4de4d666688686646de4d668688886646de4d668628886646dd4d668688886646d94d666688686646d9dd646666666694d9dd4d6666664699d9dddd6466669499d9dddd4d44449999d9dddddddd999999d9dddddddddddddddd`);
+}
+namespace palettes {
+    //% fixedInstance
+    export const Cool_Colors = hex`00000016171a7f0622d62411ff8426ffd100fafdffff80a4ff267494216a43006723497568aed4bfff3c10d275007899002859`;
+}
 ```
-#000000
```

Comment:
Asset tool won't read color values from `.hex` file in this format. No `#` prefix.